packages/db/src/core/cli/migration-queries.ts

```diff
@@ -451,7 +451,7 @@ async function getDbCurrentSnapshot(
 
 		return JSON.parse(res.snapshot);
 	} catch (error: any) {
-		if (error.code === 'SQLITE_UNKNOWN') {
+		if (error.code === 'SQLITE_UNKNOWN' || (error.code === 'SQLITE_ERROR' && error.rawCode === 1)) {
```

Comment:
It's worth leaving a comment here and explaining the check. The comment should explain:
- why `SQLITE_ERROR`?
- why `rawCode === 1`?

Reply:
Ok, I went to the source code to understand the error codes. It turns out the `rawCode` adds nothing here; it is just the numeric value of the enum while `code` is the name of the value. `code: 'SQLITE_ERROR'` will always come with `rawCode: 1`.

Also, `SQLITE_ERROR` is a generic error code used for multiple things. We'll have to check the message to ensure we only capture the error code we care about.

We'll still need to check for both codes because the library throws two errors when reading a non-existing table:
- Connected to a remote DB it throws with the code `SQLITE_UNKNOWN` and message `SQLITE_UNKNOWN: SQLite error: no such table: some_table`
- Connected to an in-memory or a local file DB it throws with the code `SQLITE_ERROR` and message `SQLITE_ERROR: no such table: some_table`

I'll change the code to properly reflect that.